products/iap/terraform.yaml

```diff
@@ -28,7 +28,7 @@ overrides: !ruby/object:Overrides::ResourceOverrides
       - !ruby/object:Provider::Terraform::Examples
         name: "iap_project"
         primary_resource_id: "project_service"
-        primary_resource_name: "fmt.Sprintf(\"tf-test%s\", context[\"random_suffix\"])"
+        primary_resource_name: "fmt.Sprintf(\"tf-test-tf-test%s\", context[\"random_suffix\"])"
```

Comment:
looks like this will actually not need a change because the example uses project_id = "tf-test%{random_suffix}" instead of var["project_id"]